### DISCUSSION TITLE
[Feature]a new mode for human pose annotaion

### DISCUSSION THREAD

**Discussion by heizie** [wkentaro/labelme#1118] [Ideas / Feature requests]
**Is your feature request related to a problem? Please describe.**
the usual annotation tools are not support to point-weis annotation. But your tool could have a great potential to deal with it. But still, some features could be provided.

**Describe the solution you'd like**
1. it is difficult to delete a point, unless we undo it, or delete it from json
2. if we annotate for huge amount of dataset, it is necessary to let the user define, which should be fix for the next time. UI intergration would be nice. could have a option to click just on the flag window
    example 1: I'll need to fix a label (maybe not just for human pose), then each click is human human human.
                   2: I'll need to fix the label and the group id, in order to annotate the same thing with same id in different frames.
3. annotate for something with some kind of sequence, like head->left shoulder-> right_shoulder..... _*i think this could have lower priority_
4. seperate videos with specific time gap. like I'll take every 5th frame from the videos. (this is quite helpful in my side. of course. this could be solve with something else. But when it is intergrated, it would be quite nice)

**Describe alternatives you've considered**
the point 4. use ffmpeg (it'll missed some frames) or use opencv

**Comment by ryouchinsa** [comment]
RectLabel is an offline image annotation tool for object detection and segmentation.
Although this is not an open source program, with RectLabel you can read/write/export keypoints in the YOLO format.
https://rectlabel.com/help#xml_to_yolo

```
class_index center_x center_y width height x1 y1 v1 x2 v2 y2 x3 y3 v3 ...
0 0.545230 0.616880 0.298794 0.766239 0.522073 0.309332 2 0.540170 0.293193 2 0.499589 0.296503 2 ...
```

A visibility flag v defined as v=0: not labeled (in which case x=y=0), v=1: labeled but not visible, and v=2: labeled and visible.
https://cocodataset.org/#format-data

![yolo_polygon](https://user-images.githubusercontent.com/1954306/209942647-fa9565e0-d4cf-42c2-a382-a8043465765b.jpg)
